contracts/truefi2/TrueCreditAgency.sol

```diff
@@ -317,17 +327,25 @@ contract TrueCreditAgency is UpgradeableClaimable, ITrueCreditAgency {
     }
 
     function currentRate(ITrueFiPool2 pool, address borrower) external view returns (uint256) {
-        return _currentRate(riskPremium.add(utilizationAdjustmentRate(pool)), creditScoreAdjustmentRate(pool, borrower));
+        return
+            _currentRate(
+                riskPremium.add(utilizationAdjustmentRate(pool)).add(getSecuredRate(pool)),
+                creditScoreAdjustmentRate(pool, borrower)
```

Comment:
I'd propose a following order:
`securedRate.add(riskPremium).add(util...).add(credit...)`

Reply:
And then apply the same order in other places where sum stuff.